src/components/FeatureViews/AdminPortal/BulkUploadModal.js

```diff
@@ -0,0 +1,523 @@
+                                </ResponsiveButton>
+                            </Grid>
+                        </Grid>
+                    </Grid>
+                );
+            case 1:
+                return (
+                    <>
+                        {loading ? (
+                            <Grid conatiner className={classes.modalStyle}>
+                                <Loading small={true} />
+                            </Grid>
+                        ) : (
+                            <Grid container className={classes.modalStyle}>
+                                <Grid item style={{ padding: '2em' }} xs={12}>
+                                    <Typography
+                                        className={classes.modalTypography}
+                                        variant='h3'
+                                    >
+                                        Upload {template}
+                                    </Typography>
+
+                                    <Typography
+                                        align='left'
+                                        className={classes.modalTypography}
+                                        variant='h6'
+                                    >
+                                        After you fill out the template, please
+                                        upload your file here. You can only
+                                        upload one file at a time.
+                                    </Typography>
+
+                                    <div>
+                                        <ResponsiveButton
+                                            onClick={upload}
+                                            disabled={disableUploadBtn}
+                                            variant='contained'
+                                        >
+                                            Select File
+                                        </ResponsiveButton>
+                                        <input
+                                            id='xml-upload'
+                                            hidden
+                                            type='file'
+                                            multiple={false}
+                                            accept='application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
+                                            onChange={handleSelectFile}
+                                        />
+                                        {fileName ? (
+                                            <>
+                                                <p>
+                                                    {fileName}{' '}
+                                                    <button
+                                                        onClick={removeUpload}
+                                                    >
+                                                        X
```

Comment:
@CalvinFronda use the [close icon](https://material-ui.com/components/material-icons/) from material ui and replace `<button/>` with `<IconButton/>` from material ui